validation/static/index.js

```diff
@@ -25,8 +25,36 @@ document.addEventListener('DOMContentLoaded', () => {
 
                 if (judgement === 'yes') {
                     button.setAttribute("class", "button button--success-solid translation-judgement-accuracy-yes")
+                    
+                    const noButtons =  document.getElementsByClassName("translation-judgement-accuracy-no");
+                    for (let b of noButtons) {
+                        if (b.dataset.phraseId === phraseId) {
+                            b.setAttribute("class", "button button--fail translation-judgement-accuracy-no")
```

Comment:
You may want to look into [`b.classList`](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#examples) instead of using `b.setAttribute()`, e.g.,

```suggestion
                            b.classList.remove("button--success")
                            b.classList.add("button--fail")
```

Reply:
I like this better--fixed all throughout this file!